ush/forecast_postdet.sh

```diff
@@ -1038,7 +1038,24 @@ GOCART_postdet() {
       rm -f "${COM_CHEM_HISTORY}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4"
     fi
 
-    ${NLN} "${COM_CHEM_HISTORY}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4" \
-           "${DATA}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4"
+    #To Do: Temporarily removing this as this will crash gocart, adding copy statement at the end 
+    #${NLN} "${COM_CHEM_HISTORY}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4" \
+    #       "${DATA}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4"
   done
 }
+
+GOCART_out() {
+  echo "SUB ${FUNCNAME[0]}: Copying output data for GOCART"
+
+  # Copy gocart.inst_aod after the forecast is run (and successfull)
+  # TO DO: this should be linked but there were issues where gocart was crashing if it was linked
+  #${NCP} "${DATA}/gocart.inst_aod.*" "${COM_CHEM_HISTORY}/"
+  if (( $( ls -1 "${DATA}/gocart.inst_aod."* 2> /dev/null | wc -l) > 0 )); then
+    for file in $(ls "${DATA}/gocart.inst_aod."*) ; do
+      ${NCP} "${file}"  "${COM_CHEM_HISTORY}/"
+    done
+  fi
```

Comment:
This works for me on WCOSS:
```suggestion
  for fhr in ${FV3_OUTPUT_FH}; do
    if (( fhr == 0 )); then continue; fi
    local vdate=$(date --utc -d "${current_cycle:0:8} ${current_cycle:8:2} + ${fhr} hours" +%Y%m%d%H)
    ${NCP} "${DATA}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4" \
      "${COM_CHEM_HISTORY}/gocart.inst_aod.${vdate:0:8}_${vdate:8:2}00z.nc4"       
  done
```

Note: I originally tried to test on Orion and the forecast failed pretty quickly. I think there may be something still wrong there.

Reply:
I'll try to run the branch as in on orion before implementing this change.   I don't remember issues running on orion and there were stmp issues overnight so hopefully it's simply was a space issue.